paracompact + locally compact + connected implies exhaustible by compacts

Moniker1998 opened the pull request:
See #1214

prabau commented:
Reading your proof.  I agree that the constructed $\bigcup_n L_n$ is an open set and is sigma-compact.

But it's not clear to me why it's also a closed set.  Maybe one would have to show that after constructing the union of the $L_n$, one can take a point outside of that union and repeat the construction, and one would get a another open sigma-compact set, disjoint from the first one?  Maybe rephrase the argument in terms of some equivalence relation...

Not sure actually what the next to last sentence in the big paragraph in the middle (starting with "If $y\in U_i$ and ...") is saying in this regard.

yhx-12243 commented:
In fact the condition of paracompact can be weaken to para-Lindelöf, see [comment](https://math.stackexchange.com/questions/3246970/connected-locally-compact-paracompact-hausdorff-space-is-exhaustible-by-compac#comment-10792739) in that mathse post.

Moniker1998 commented:
@prabau @yhx-12243 I've addressed your issues Patrick, and by suggestion of yhx, i I've replaced paracompact by para-Lindelöf, and since there is really nothing stopping me, I've changed locally compact to locally Lindelöf as well, which might some time arrive at pi-base.

prabau commented:
@Moniker1998 Thanks for the clear reorganization of the post.  Very readable.

About the main result (If X is locally Lindelöf and para-Lindelöf, then X is a union of pairwise disjoint clopen Lindelöf sets):

The proof shows that each $x\in X$ is in some clopen Lindelof set.  But it still does not show that the whole space can be partitioned into these sets.  I.e., starting with some $x$ one constructs $L=\bigcup_n L_n$ clopen Lindelof.  Then one takes a $z\notin L$ and repeat the construction to get a clopen Lindelof set $M=\bigcup_n M_n$$ around $z$.  But it could be that $M$ intersects $L$.  Need to think more on this.

@yhx-12243 fyi

prabau commented:
Regarding the notion of "locally Lindelöf", this should probably first be discussed in a separate issue, but I think it would be useful to add the definition in pi-base.  And I am sure there are more results out there that make use of the notion, even if not by name.

But as for many "local properties", there are different variants, depending on whether one requires a local base of nbhds with the property around each point, or just one nbhd with the property around each point.  Both notions are useful.
See https://github.com/pi-base/data/wiki/Conventions-and-Style#local-properties for the general convention used in pi-base to distinguish between the two.

In particular, we have
- Weakly locally compact (P23)
- Locally compact (P130)

(Each paper usually has to define what they mean exactly by "locally compact".  But note that the term "weakly locally compact" is not just a pi-base invention.  It does appear in a few cases in the literature (see references in https://en.wikipedia.org/wiki/Locally_compact_space, where it is also mentioned).  At least it is clear and unambiguous.  That convention makes it easy to distinguish between the two.

Similarly, there will be:
- Weakly locally Lindelöf
- Locally Lindelöf

For the mathse post, the notion is the first one.

In any case, I think it would be useful to edit the post to use this convention, i.e. rename "locally compact" to "weakly locally compact" and "locally Lindelöf" to "weakly locally Lindelöf" in your answer.  That will keep things clear, and will match with what will be in pi-base eventually.

Opinions?

Moniker1998 commented:
> @Moniker1998 Thanks for the clear reorganization of the post. Very readable. (I fixed a minor thing in the post.)
> 
> About the main result (If X is locally Lindelöf and para-Lindelöf, then X is a union of pairwise disjoint clopen Lindelöf sets):
> 
> The proof shows that each x ∈ X is in some clopen Lindelof set. But it still does not show that the whole space can be partitioned into these sets. I.e., starting with some x one constructs L = ⋃ n L n clopen Lindelof. Then one takes a z ∉ L and repeat the construction to get a clopen Lindelof set M = ⋃ n M n $ around z . But it could be that M intersects L . Need to think more on this.
> 
> Corollary 1 is valid no matter what, but it would be good to prove the theorem also.
> 
> @yhx-12243 fyi

I think for that one would possibly have to define locally Lindelöf as a space for which every $x\in X$ has open neighbourhood $U$ such that $\overline{U}$ is Lindelöf, because then you can just use an argument that the set $\bigcup_n L_n$ is the set of all points which are chained with $x$ by some finite amount of $U_{x_i}$, that is $x\in U_{x_1}, U_{x_i}\cap U_{x_{i+1}}\neq\emptyset$ and $y\in U_{x_n}$.

I don't think it holds in general and we would need some space which is para-Lindelöf and every point has Lindelöf neighbourhood but there is a point whose every open neighbourhood has non-Lindelöf closure.

Moniker1998 commented:
> @Moniker1998 Thanks for the clear reorganization of the post. Very readable. (I fixed a minor thing in the post.)
> 
> About the main result (If X is locally Lindelöf and para-Lindelöf, then X is a union of pairwise disjoint clopen Lindelöf sets):
> 
> The proof shows that each x ∈ X is in some clopen Lindelof set. But it still does not show that the whole space can be partitioned into these sets. I.e., starting with some x one constructs L = ⋃ n L n clopen Lindelof. Then one takes a z ∉ L and repeat the construction to get a clopen Lindelof set M = ⋃ n M n $ around z . But it could be that M intersects L . Need to think more on this.
> 
> Corollary 1 is valid no matter what, but it would be good to prove the theorem also.
> 
> @yhx-12243 fyi

Yeah, I get the issue with my proof, and I fixed it. Thanks

Moniker1998 commented:
> In any case, I think it would be useful to edit the post to use this convention, i.e. rename "locally compact" to "weakly locally compact" and "locally Lindelöf" to "weakly locally Lindelöf" in your answer. That will keep things clear, and will match with what will be in pi-base eventually.

I don't think thats really necessary, naming conventions don't always have to agree with each other.

Moniker1998 commented:
> But as for many "local properties", there are different variants, depending on whether one requires a local base of nbhds with the property around each point, or just one nbhd with the property around each point. Both notions are useful.
See https://github.com/pi-base/data/wiki/Conventions-and-Style#local-properties for the general convention used in pi-base to distinguish between the two.

Of course there are more than just two notions of locally compact space, and same should be true for locally Lindeof (I always found that strange in pi-base...). Moreover I don't think its necessary to accept two new definitions, one of them won't be used then it seems.

prabau commented:
Nice fix to the proof.  Thanks.

prabau commented:
Nice consequence: will be able to derive that two more spaces, including S153 (Open long ray), is not para-Lindelof.

prabau commented:
Looks good overall.  I'll make a final suggestion to explain what is meant by the term used here that is not yet in pi-base, and to follow the pi-base convention.

prabau commented on theorems/T000698.md:
```diff
@@ -0,0 +1,15 @@
+    name: Answer to "Connected, locally compact, paracompact Hausdorff space is exhaustible by compacts"
+---
+
+In {{mathse:5028016}} it was shown that a {P36} locally Lindelöf {P105} space is {P18}, and {P23} implies locally Lindelöf.
```
```suggestion
Call a space *weakly locally Lindelöf* if every point has a neighborhood that is Lindelöf.

In {{mathse:5028016}} it is shown that a {P36} weakly locally Lindelöf {P105} space is {P18}, and {P23} implies weakly locally Lindelöf.
```
